test/mocks/delinkedCalculation.js

```diff
@@ -41,11 +45,14 @@ const mockDelinkedCalculation2 = {
   referenceAmount: 12503.38,
   progressiveReduction: 25006.76,
   totalDelinkedPayment: 12503.38,
-  paymentAmountCalculated: 6251.6
+  paymentAmountCalculated: 6251.6,
+  datePublished: new Date(),
+  updated: new Date()
 }
 const mockDelinkedCalculation3 = {
   applicationId: 1810907,
   calculationId: 123456787,
+  calculationReference: 123456787,
```

Comment:
if we were to make the change to databases above then we have no need for a calculationReference here.